utils/check-incremental

```diff
@@ -19,74 +19,73 @@
 #
```

Comment:
@gribozavr No, that was not intentional. Fixed and pushed. Thanks for noticing! :-)